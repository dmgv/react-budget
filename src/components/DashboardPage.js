import React, { Fragment } from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import ExpensesSummary from "./ExpensesSummary";

const DashboardPage = () => (
  <Fragment>
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList />
  </Fragment>
);

export default DashboardPage;
