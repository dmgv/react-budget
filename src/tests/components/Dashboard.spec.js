import React from 'react'
import {shallow} from 'enzyme'
import DashboardPage from '../../components/DashboardPage'

test('Should have a dashboard',() => {
  const wrapper = shallow(<DashboardPage />);
  expect(wrapper).toMatchSnapshot();
})
