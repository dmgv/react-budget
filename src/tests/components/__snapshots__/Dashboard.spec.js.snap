// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should have a dashboard 1`] = `
<React.Fragment>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</React.Fragment>
`;
